sdk-setup/src/mb2

```diff
@@ -320,9 +320,10 @@ COMMANDS
             root access to the device.
 
         --rsync
-            Uses 'rsync' to copy the installed files under '/opt/sdk/<name>'
-            without building/installing RPM package(s).  Does not require root
-            access to the device.  Incompatible with '--all'.
+            This method is special.  Uses 'rsync' to copy the installed files
```

Comment:
I don't think "This method is special" is useful information. Instead, perhaps start by stating "This method does not deal with RPM packages. Instead it ..."

Reply:
I want to clearly distinguish this method from the others. The fact that it does not deal with RPM packages has other implications. Only all these together make this method so special, and this is something one may not easily realize. Hence the tl;dl message. Also, the main description only deals with this method as little as possible - otherwise if would be full of "except for" and "only if".

Now I noticed I left "The '--all' option is not available when the '--rsync' method is used." in the main description - this is to be moved here too.